docs/lightning/04-salesforce-lambdas/04-contact-channel-analytics.md

```diff
@@ -95,7 +95,7 @@ Cloudformation stack:
 
 4. Under the "Secret value" tab, select "Retrieve secret value" and then "Edit".
 
-5. For the **CloudFrontPrivateKey** field, copy and paste the modified contents of the private key .pem file. For the **CloudFrontAccessKeyID** field, copy and paste the **Access Key Id** you recorded above. Your Secrets Manager Secret should look like the following:
+5. For the **CloudFrontPrivateKey** field, copy and paste the modified contents of the private key .pem file. For the **CloudFrontAccessKeyID** field, copy and paste the **public key ID** you recorded above. Your Secrets Manager Secret should look like the following:
```

Comment:
It should be `public key ID` as per mentioned under this link : https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/private-content-trusted-signers.html

Reply:
I believe I added this change with a previous commit I made - https://github.com/amazon-connect/amazon-connect-salesforce-cti/commit/0974657a202fdc91f6bfa516ffb4aea5e53799f1, but good find!

Reply:
yes, updated